run-tests.sh

```diff
@@ -24,6 +24,6 @@
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
 pydocstyle invenio_workflows && \
-sphinx-build -qnNW docs docs/_build/html && \
+sphinx-build -qnN docs docs/_build/html && \
 python setup.py test && \
-sphinx-build -qnNW -b doctest docs docs/_build/doctest
+sphinx-build -qnN -b doctest docs docs/_build/doctest
```

Comment:
There's a bit of a mess with the deps, you might need to change the `setup.py` and `tox.ini` files.